DxDispatch/src/dxdispatch/Device.cpp

```diff
@@ -17,12 +17,23 @@ static bool DebugMessageCallback(void* context, void* commandList, DWORD message
 #else
 static void __stdcall DebugMessageCallback(D3D12_MESSAGE_CATEGORY cat, D3D12_MESSAGE_SEVERITY sev, D3D12_MESSAGE_ID id, LPCSTR message, void* context)
 {
-    LogError(message);
+    auto fmtMessage = fmt::format("{} {} {} {}", cat, id, context, message);
+    if( (D3D12_MESSAGE_SEVERITY_INFO == sev) ||
+        (D3D12_MESSAGE_SEVERITY_MESSAGE == sev) ||
+        (D3D12_MESSAGE_SEVERITY_WARNING == sev))
+    {
+        LogInfo(fmtMessage);
+    }
+    else
+    {
+        LogError(fmtMessage);
+    }
 }
 #endif
 
 Device::Device(
     IAdapter* adapter, 
+    D3D_FEATURE_LEVEL featureLevel,
```

Comment:
>D3D_FEATURE_LEVEL [](http://example.com/codeflow?start=4&length=17)

@jstoecker should I update the default D3D12_COMMAND_LIST_TYPE to be D3D12_COMMAND_LIST_TYPE_COMPUTE so they work with D3D_FEATURE_LEVEL_1_0_CORE devices by default

Reply:
I think that makes sense.